eb_hooks.py

```diff
@@ -335,6 +335,20 @@ def pre_configure_hook_LAMMPS_aarch64(self, *args, **kwargs):
         raise EasyBuildError("LAMMPS-specific hook triggered for non-LAMMPS easyconfig?!")
 
 
+def pre_configure_hook_atspi2core_filter_ld_library_path(self, *args, **kwargs):
+    """
+    pre-configure hook for at-spi2-core:
+    - instruct GObject-Introspection's g-ir-scanner tool to not set $LD_LIBRARY_PATH
+      when EasyBuild is configured to filter it
```

Comment:
let's mention https://github.com/EESSI/software-layer/issues/196 in here too?

Reply:
done